srtcore/handshake.cpp

```diff
@@ -141,9 +141,22 @@ std::string RequestTypeStr(UDTRequestType rq)
 {
     if (rq >= URQ_FAILURE_TYPES)
     {
-        SRT_REJECT_REASON rej = RejectReasonForURQ(rq);
-        int id = rej;
-        return std::string("ERROR:") + srt_rejectreason_name[id];
+        std::ostringstream rt;
+        rt << "ERROR:";
+        int id = RejectReasonForURQ(rq);
+        if (id < SRT_REJ_E_SIZE)
+            rt << srt_rejectreason_name[id];
+        else if (id < SRT_REJC_USER)
+        {
+            if (id < SRT_REJC_SERVER)
```

Comment:
`<=` instead of `<`?

Reply:
I thought your previous comment defined that, e.g., `SRT_REJC_SERVER` exact value is included in the range for server codes...

Reply:
Right, all good. The function returns "unknown" if `id < SRT_REJC_SERVER`